src/core/Akka.Remote/EndpointManager.cs

```diff
@@ -730,7 +730,6 @@ protected void Accepting()
                         _endpoints.MarkAsQuarantined(gotuid.RemoteAddress, gotuid.Uid,
                             Deadline.Now + _settings.QuarantineDuration);
                         _eventPublisher.NotifyListeners(new QuarantinedEvent(gotuid.RemoteAddress, gotuid.Uid));
-                        Context.Stop(pass.Endpoint);
```

Comment:
if policy is WasGated Context.Stop(pass.Endpoint) will cause null reference exception
https://github.com/akka/akka/blob/master/akka-remote/src/main/scala/akka/remote/Remoting.scala#L704